File: List 3/data/array/1.cs
/*
Crie um array de inteiros com 5 posições. Atribua valores manualmente e imprima todos usando
um for tradicional.
*/

int[] numbers = new int[5];
int e; 

for(int i = 0; i < numbers.Length; i++)
{
    Console.WriteLine($"Please, number {i + 1}: ");
    e = int.Parse(Console.ReadLine());
    numbers[i] = e;
}

for(int i = 0; i < numbers.Length; i++)
{
    Console.WriteLine(numbers[i]);
}

File: List 3/data/array/1.php
<?php

/*
Crie um array de inteiros com 5 posições. Atribua valores manualmente e imprima todos usando
um for tradicional.
*/

$string = [];

for ($i = 0; $i < 5; $i++) {
    $text = intval(readline("Entre um número\n"));
    $string[intval($i)] = $text;
}

foreach($string as $key){
    echo $key;
}